module/src/hooks/form/form.stories.mdx

```diff
@@ -97,7 +97,7 @@ Initial state can also be passed as a React callback. The callback will be execu
 
 This is extremely useful when a form relies on remote data that could update at any time, it allows the form to update with the latest changes from the server without losing the user's current edits by completely replacing the form data with the remote update.
 
-**NOTE:** When using an initial data function, be aware that it will be executed to provide initial data whenever it's reference updates. It's essential therefore that any initial data function is a callback with a carefully selected set of dependencies, rather than an anonymous function or constant function.
+**NOTE:** When using an initial data function, be aware that it will be executed to provide initial data whenever its reference updates. It's essential therefore that any initial data function is a callback with a carefully selected set of dependencies, rather than an anonymous function or constant function.
```

Comment:
Hero

Reply:
I can't help it, I'm a pedant over apostrophes...